dod-advana/gamechanger-web
# Task/favorite groups

Description: 
-Creates a "Groups" section in the user dashboard for creating and managing groups of favorites. Currently limited to favorite documents.

Ticket: https://jira.di2e.net/browse/UOT-110601

Testing Instructions:
- From "Groups" tab on the User Dashboard
     -Create groups from button top right
          -Name can't be repeated
-From "Documents" section in the favorites tab of the User Dashboard
     -Add favorites to a group from button top right
          -Groups should only be able to take 5 documents. Error shows is 5 exceeded
     -Removing a favorite should also remove the item from any groups
- From "Groups" tab on the User Dashboard
     -Remove documents from group with star in top right corner
     -View favorites' info on card once clicked
          -View favorite cummary by clicking on speech bubbly icon on the bottom
          -Open document in PDF viewer with "Open" button (doesn't work on local)
          -Open details page with "Details" button
     -Exporting groups from button in top right corner of groups. Currently uses a combination of the favorites' saved search texts
     -Delete groups from button in top right
